examples/create_rosa_cluster/create_rosa_sts_cluster/byo_oidc_cluster/output.tf

```diff
@@ -0,0 +1,11 @@
+output cluster_id {
+  value = ocm_cluster_rosa_classic.rosa_sts_cluster.id
+}
+
+output oidc_thumbprint {
+  value = ocm_cluster_rosa_classic.rosa_sts_cluster.sts.thumbprint
+}
+
+output oidc_endpoint_url {
```

Comment:
not sure that it needs to be in the output file

Reply:
We can keep as a way to assure the client of what he's using. No need for it for other steps

Reply:
Removed :)